lib/src/configuration.dart

```diff
@@ -39,14 +39,19 @@ class Configuration {
   /// [readOnly] controls whether a [Realm] is opened as readonly.
   /// This allows opening it from locked locations such as resources,
   /// bundled with an application.  The realm file must already exists.
-  Configuration(List<SchemaObject> schemaObjects, {bool readOnly = false})
+  ///
+  /// [inMemoryOnly] configures [Realm] instance to be deleted after it is closed.
```

Comment:
```suggestion
  /// If [inMemoryOnly] is set, then the [Realm] instance will not be persisted, and hence deleted after it is closed.
```

Reply:
.. but maybe I'm wrong and it is actually persisted, and just gets deleted when closed? I would be surprised, but then again it wouldn't be the first time 😊

Reply:
Okay 😄